kt/godot-library/src/main/kotlin/godot/core/Rect2i.kt

```diff
@@ -1,16 +1,228 @@
+@file:Suppress("GrazieInspection")
+
 package godot.core
 
+import godot.Side
+import godot.annotation.CoreTypeHelper
+import kotlin.math.max
+import kotlin.math.min
+
+
+private typealias Point2i = Vector2i
+private typealias Size2i = Vector2i
+
 class Rect2i(
-    p_position: Vector2i,
-    p_size: Vector2i
+    p_position: Point2i,
+    p_size: Size2i
 ) {
     @PublishedApi
-    internal var _position = Vector2i(p_position)
+    internal var _position = Point2i(p_position)
     @PublishedApi
-    internal var _size = Vector2i(p_size)
+    internal var _size = Size2i(p_size)
+
+    //PROPERTIES
+    /** Return a copy of the position Vector2i
+     * Warning: Writing position.x = 2 will only modify a copy, not the actual object.
+     * To modify it, use position().
+     * */
```

Comment:
I suspect this comment does not come from godot. It's our own? If so I'd properly reference the function `position()` using proper kdoc syntax (I believe in this case it would be a simple wrapping in `[]`. So `[position()]` should do it.

Also `Warning` could be bold. and to properly space the "title" from the body you'd need two line breaks not just one.

Reply:
Also wrap code samples in with a backtick `